src/slash-bedrock/share/brl-fetch/distros/funtoo

```diff
@@ -0,0 +1,85 @@
+#!/bedrock/libexec/busybox sh
+#
+# Funtoo Linux bootstrap support
+#
+#      This program is free software; you can redistribute it and/or
+#      modify it under the terms of the GNU General Public License
+#      version 2 as published by the Free Software Foundation.
+#
+# Copyright (c) 2022-2022 Nicolas Lorin <androw95220@gmail.com>
+#
+
+# shellcheck source=src/slash-bedrock/libexec/brl-fetch
+. /bedrock/share/common-code
+trap 'fetch_abort "Unexpected error occurred."' EXIT
+
+check_supported() {
+        true
```

Comment:
This (poorly named) function returning true indicates there's a Bedrock maintainer for the distro, but [there is no maintainer at this point](https://bedrocklinux.org/0.7/distro-compatibility.html).  If [you want to become a Bedrock Funtoo maintainer we can certainly discuss it](https://bedrocklinux.org/contributing.html#distro-maintainer), but until that's resolved this should be `false`.

Reply:
Switch to false at the moment